io/www/test/fragment/mocks/fragment-ah-de_DE-corrupted.json

```diff
@@ -0,0 +1,282 @@
+{
```

Comment:
nit: can you make that mock a bit smaller :) ?

Reply:
done